app/admin/store_products.rb

```diff
@@ -1,8 +1,8 @@
 ActiveAdmin.register Product do
   menu parent: I18n.t('activeadmin.titles.stores')
 
-  permit_params :store_id, :name, :category_id, :price, :link,
-                :email, :gender, :age, :novelty, :status
+  permit_params :store_id, :name, :category_id, :price, :description,
+                :link, :email, :gender, :age, :novelty, :status
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.